shopfloor_mobile/static/wms/src/settings/settings.js

```diff
@@ -15,18 +36,23 @@ export var SettingsControlPanel = Vue.component("settings-control-panel", {
                     <v-col class="text-center" cols="12">
                         <v-btn @click="$router.push({'name': 'profile'})">
                             <v-icon>mdi-account-cog</v-icon>
-                            <span>{{ $t("screen.settings.profile.name") }}</span>
+                            <span>{{ $t("screen.settings.profile.name") }} - {{ profile_name }}</span>
                         </v-btn>
                     </v-col>
                 </v-row>
                 <v-row align="center">
                     <v-col class="text-center" cols="12">
                         <v-btn @click="$router.push({'name': 'language'})">
                             <v-icon>mdi-flag</v-icon>
-                            <span>{{ $t("screen.settings.language.name") }}</span>
+                            <span>{{ $t("screen.settings.language.name") }} - {{ this.active_language }}</span>
                         </v-btn>
                     </v-col>
                 </v-row>
+                <v-row align="center">
+                    <v-col class="text-center" cols="12">
+                        <btn-action @click="logout()">{{ $t('screen.settings.profile.action.logout') }}</btn-action>
```

Comment:
```suggestion
                        <btn-action @click="logout()">{{ $t('app.action.logout') }}</btn-action>
```
and move the translation in i18n file as this is a generic action that could be done from everywhere.

Reply:
Moved